cms/jinja2/templates/pages/statistical_article_page--related_data.html

```diff
@@ -0,0 +1,52 @@
+{% extends "templates/base_page.html" %}
+
+{% from "components/document-list/_macro.njk" import onsDocumentList %}
+{% from "components/hero/_macro.njk" import onsHero %}
+{% from "components/pagination/_macro.njk" import onsPagination %}
+
+{% block header_area %}
+    {# fmt:off #}
+    {{
+        onsHero({
+            "topic": _("Related data"),
+            "title": page.related_data_display_title,
+            "variants": 'grey',
+            "text": page.summary,
+            "officialStatisticsBadge": page.is_accredited,
+            "wide": true,
+            "detailsColumns": 12,
+            "censusLogo": page.is_census,
+            "breadcrumbs": {
+                "ariaLabel": _("Breadcrumbs"),
+                "itemsList": breadcrumbs(page, include_self=True),
+            }
+        })
+    }}
+    {# fmt:on #}
+{% endblock %}
+
+{% block main %}
+
+    {# fmt:off #}
+    {{
+        onsDocumentList({
+        "documents": paginated_datasets
+        })
+    }}
+    {# fmt:on #}
```

Comment:
nitpick:

```suggestion
    {{ onsDocumentList({"documents": paginated_datasets}) }}
```

should be enough

or, if going the formatted way:

```suggestion
    {# fmt:off #}
    {{
        onsDocumentList({
            "documents": paginated_datasets
        })
    }}
    {# fmt:on #}
```